api/FocusEvent.json

```diff
@@ -104,40 +104,40 @@
           "spec_url": "https://w3c.github.io/uievents/#dom-focusevent-relatedtarget",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "26"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "26"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "48"
```

Comment:
This came from https://github.com/mdn/browser-compat-data/pull/1719 which was wiki migration, so not trusting this.